eclass/multilib.eclass

```diff
@@ -233,8 +233,6 @@ get_libname() {
 		*-cygwin*)       libname="dll.a";; # import lib
```

Comment:
what about this one?

Reply:
I kept this because there's a possibility of cross, but actually, let's just kill it off.